pkg/webui/console/containers/dev-eui-component/index.js

```diff
@@ -94,7 +94,11 @@ const DevEUIComponent = props => {
         ) {
           const generatedId = `eui-${values.authenticated_identifiers.dev_eui.toLowerCase()}`
           setFieldValue('target_device_id', generatedId)
+
+          return generatedId
```

Comment:
It was a way of preventing it from getting into both ifs, but I think I found a better way

Reply:
You don't have to even make the condition. You can just set the field in both paths:
```js
setFieldValue('target_device_id', generatedId)
setFieldValue('ids.device_id', generatedId)
```
The irrelevant one will be stripped when submitting anyway.

Reply:
Actually I just realized why I put that if there. If we are setting the dev eui in claiming, it will be stored in `authenticated_identifiers`, while if it's regular registration it will be stored in `ids`. It's two separate fields. In this if, I have to `setFieldValue('ids.device_id', generatedId)`, if it's stored in `ids` and `setFieldValue('target_device_id', generatedId)` if it's stored in `authenticated_identifiers`.